modules/tensor_mechanics/test/tests/thermal_expansion/constant_expansion_coeff.i

```diff
@@ -36,7 +36,7 @@
   [./TensorMechanics]
     [./Master]
       [./all]
-        strain = SMALL
+        strain = FINITE
```

Comment:
Why the change from small? The thermal eigenstrains are all based on linear assumptions, so my guess is that it was made small strain originally because it's easier to directly compare against a known solution.